includes/option-types/form-builder/extends/class-fw-option-type-form-builder-item.php

```diff
@@ -67,12 +67,12 @@ public function get_value_from_item( $value ) {
 	}
 
 	/**
-	 * Search in '/extensions/forms/{builder_type}/items/{item_type}/options/options.php'
+	 * Search in '/extensions/forms/{builder_type}/items/{item_type}/options.php'
```

Comment:
Well, that might be a breaking change. @ViorelEremia be careful with that. There may be developers that already have overrided t `options/options.php` and may expect it to be that way.

Reply:
So far I am the only one [talking about this improvement](https://github.com/ThemeFuse/Unyson/issues/1738), I don't think it's gonna be a problem. And the change to the new version is super easy anyway.

Reply:
I am overriding original

Reply:
So it's OK then, we can skip that change, I was hoping to get that back to "normal".

Reply:
Actually im overriding extensions\forms\extensions\contact-forms\shortcodes\contact-form\options.php

still digging trough your changes